src/main/java/racingcar/view/ProgressingConsole.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.view;
+
+import java.util.Objects;
+
+public class ProgressingConsole {
+
+    private boolean isAbNormalCarsName;
+    private boolean isAbNormalRacingTime;
+
+    public ProgressingConsole() {
+        this.isAbNormalCarsName = true;
+        this.isAbNormalRacingTime = true;
```

Comment:
```suggestion
        this(true, true)
```
메서드 체이니으로 표현이 가능합니다.